### PR TITLE
Ceph: updates CephCluster CRD doc to include monitoring options

### PR DESCRIPTION
**Pull request by umangachapagain**
Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

**Description of your changes:**
Updating CephCluster CRD documentation

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](https://github.com/rook/rook/blob/master/CONTRIBUTING.md#comments)
- [x] Add the flag for skipping the CI if this PR does not require a build. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for more details.

[skip ci]

### CODE REVIEW

File: Documentation/ceph-cluster-crd.md [hunk 1]
```diff
@@ -112,6 +112,14 @@ If this value is empty, each pod will get an ephemeral directory to store their
   * `urlPrefix`: Allows to serve the dashboard under a subpath (useful when you are accessing the dashboard via a reverse proxy)
   * `port`: Allows to change the default port where the dashboard is served
   * `ssl`: Whether to serve the dashboard via SSL, ignored on Ceph versions older than `13.2.2`
+* `monitoring`: Settings for monitoring Ceph using Prometheus. To enable monitoring on your cluster see the [monitoring guide](Documentation/ceph-monitoring.md#prometheus-alerts).
+  * `enabled`: Whether to enable prometheus based monitoring for this cluster
+  * `rulesNamespace`: Namespace to deploy prometheusRule. If empty, namespace of the cluster will be used.
+  
+    Recommended:
+
+    * If you have a single rook-ceph cluster, set the rulesNamespace to the same namespace as the cluster or keep it empty.
```
Comment (galexrt) [inline Documentation/ceph-cluster-crd.md]: What about changing `rook-ceph cluster` / `rook-ceph clusters` to `Rook Ceph cluster` and `Rook Ceph clusters`

File: Documentation/ceph-cluster-crd.md [hunk 2]
```diff
@@ -112,6 +112,14 @@ If this value is empty, each pod will get an ephemeral directory to store their
...
+    * If you have multiple rook-ceph clusters in the same k8s cluster, choose the same namespace (ideally, namespace with prometheus deployed) to set rulesNamespace for all the clusters. Otherwise, you will get duplicate alerts with multiple alert definitions.
```
Comment (galexrt) [inline Documentation/ceph-cluster-crd.md]: Please use backticks to surround the parameters in this text to make it easier to read and understand (e.g., backticks around `rulesNamespace`

File: Documentation/ceph-cluster-crd.md (hunk 1 above)
Comment (umangachapagain) [reply Documentation/ceph-cluster-crd.md]: ack.

File: Documentation/ceph-cluster-crd.md (hunk 2 above)
Comment (umangachapagain) [reply Documentation/ceph-cluster-crd.md]: ack

File: Documentation/ceph-cluster-crd.md [hunk 3]
```diff
@@ -112,6 +112,12 @@ If this value is empty, each pod will get an ephemeral directory to store their
   * `urlPrefix`: Allows to serve the dashboard under a subpath (useful when you are accessing the dashboard via a reverse proxy)
   * `port`: Allows to change the default port where the dashboard is served
   * `ssl`: Whether to serve the dashboard via SSL, ignored on Ceph versions older than `13.2.2`
+* `monitoring`: Settings for monitoring Ceph using Prometheus. To enable monitoring on your cluster see the [monitoring guide](Documentation/ceph-monitoring.md#prometheus-alerts).
+  * `enabled`: Whether to enable prometheus based monitoring for this cluster
+  * `rulesNamespace`: Namespace to deploy prometheusRule. If empty, namespace of the cluster will be used.
+      Recommended:
+    * If you have a single Rook Ceph cluster, set the `rulesNamespace` to the same namespace as the cluster or keep it empty.
+    * If you have multiple Rook Ceph clusters in the same k8s cluster, choose the same namespace to set `rulesNamespace` for all the clusters (ideally, namespace with prometheus deployed). Otherwise, you will get duplicate alerts with duplicate alert definitions.
```
Comment (galexrt) [inline Documentation/ceph-cluster-crd.md]: One last thing `k8s` -> `Kubernetes` as it is in other parts of the document

Comment (umangachapagain) [reply Documentation/ceph-cluster-crd.md]: done.

**Comment by umangachapagain** [comment]
> Please add a commit message.

Done.

**Pull request merged**